backend/src/sockets/joinRoom.js

```diff
@@ -1,11 +1,22 @@
 import games from '../domain/Games';
+import Session from '../mongo/models/Session';
+
 /**
  * Handles event when user joins a room. Adds to corresponding roomid.
  * @param {*} socket
  * @param {*} io
  */
 export function joinRoom(socket, io) {
-  socket.on('join_room', ({sessionId, name}) => {
+  socket.on('join_room', async ({sessionId, name}) => {
+
+    const finishedSession = await findSession(sessionId)
+    
+    // console.log(finishedSession.isFinished);
+    if (finishedSession && finishedSession.isFinished) {
+      io.to(socket.id).emit('skip-to-results', true)
+      console.log("here");
+    } else {
+
```

Comment:
Do we want to abstract the else functionality out? :D I feel this function is quite large